src/Tags/Livewire.php

```diff
@@ -86,4 +86,37 @@ public function scriptConfig(): string
     {
         return \Livewire\Mechanisms\FrontendAssets\FrontendAssets::scriptConfig();
     }
+    
+    /**
+     * Antlers implementation of @assets - https://livewire.laravel.com/docs/javascript#loading-assets
+     *
+     * {{ livewire:assets }}....{{ /livewire:assets }}
+     */
+    public function assets(): void
+    {
+        $html = (string) $this->parse();
+        
+        $key = md5($html);
+        
+        \Livewire\store($this->context['__livewire'])->push('assets', $html, $key);
+    }
+    
+    /**
+     * Antlers implementation of @script - https://livewire.laravel.com/docs/javascript#executing-scripts
+     *
+     * {{ livewire:script }}...{{ /livewire:script }}
+     */
+    public function script(): void
+    {
+        $html = trim((string) $this->parse());
+        
+        $key = md5($html);
+        
+        if (in_array($key, \Livewire\Features\SupportScriptsAndAssets\SupportScriptsAndAssets::$alreadyRunAssetKeys)) {
```

Comment:
why the empty `if`?

Reply:
Just lifted it from the livewire code. Makes it easier to copy paste if it changes

Reply:
That's not the code in the `scripts` code in LW though:

```php
        Blade::directive('script', function () {
            $key = static::getUniqueBladeCompileTimeKey();

            return <<<PHP
                <?php
                    \$__scriptKey = '$key';
                    ob_start();
                ?>
            PHP;
        });

        Blade::directive('endscript', function () {
            return <<<PHP
                <?php
                    \$__output = ob_get_clean();

                    \Livewire\store(\$this)->push('scripts', \$__output, \$__scriptKey)
                ?>
            PHP;
        });
```

Reply:
had them the wrong way round, thanks